docs/cue/reference/components/sources.cue

```diff
@@ -107,6 +113,12 @@ components: sources: [Name=string]: {
 		}
 
 		if features.receive != _|_ {
+			if features.receive.proxy != _|_ {
+				if features.receive.proxy.enabled {
+					proxy: configuration._proxy
+				}
+			}
+
```

Comment:
Was there a source that is "receive" that uses the proxy? I think the proxy might only be relevant for "collect" sources which initiate requests.